databricks_cli/pipelines/cli.py

```diff
@@ -177,7 +177,7 @@ def list_cli(api_client):
 
 
 @click.command(context_settings=CONTEXT_SETTINGS,
-               short_help='Resets a delta pipeline so data can be reprocessed from scratch')
+               short_help='[Deprecated] Resets a pipeline so data can be reprocessed from scratch')
```

Comment:
For better internationalization:
"so data" -> "so that data"
"scratch" -> "from the beginning"

Reply:
done